general/binaryio.hpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2010, Lawrence Livermore National Security, LLC. Produced at
+// the Lawrence Livermore National Laboratory. LLNL-CODE-443211. All Rights
+// reserved. See file COPYRIGHT for details.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability see http://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the GNU Lesser General Public License (as published by the Free
+// Software Foundation) version 2.1 dated February 1999.
+
+#ifndef MFEM_BINARYIO
+#define MFEM_BINARYIO
+
+#include "../config/config.hpp"
+
+#include <iostream>
+
+namespace mfem
+{
+
+// binary I/O helpers
```

Comment:
I'm thinking it maybe better to move these two functions inside their own namespace, e.g. `mfem::bin_io` or `mfem::io`. What do you think, @jakubcerveny?

Reply:
That's probably a good idea. I'm voting for mfem::bin_io.

Reply:
OK, I'll make the edits later today.